manager.go

```diff
@@ -229,7 +229,7 @@ type Options struct {
 // NetlinkSocket - (TC classifier programs and XDP) Netlink socket cache entry holding the netlink socket and the
 // TC filter count
 type NetlinkSocket struct {
-	sock          *netlink.Handle
+	Sock          *netlink.Handle
```

Comment:
Exporting `Sock` doesn't seem necessary for these code changes? Is it for something else?

Reply:
Hey 👋🏻 
I needed access to this `Sock` for the `TC classifiers` PR mentioned above. But you're right, this is unrelated to the PID namespace PR, moving this change to the `will/preserve-tc-env` branch.